### PR TITLE
GSoC 2024 updates

### PR DESCRIPTION
**Pull request by priyasharma1**

### CODE REVIEW

File: source/gsoc/gsoc-admin-tasks.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ contributors to your projects.
 If you want to participate with your project you now need to
 
  - understand what it means to be a mentor [1]
- - propose your project ideas. Just label your issues with gsoc2023 in Jira and
+ - propose your project ideas. Just label your issues with gsoc2024 in Jira and
    they will show up at [2]. See also [1].
```
Comment (bdelacretaz) [inline source/gsoc/gsoc-admin-tasks.md:27]: The `[2] https://s.apache.org/gsoc2011tasks` footnote is probably not useful anymore, either this page needs to be reworked to remove such precise references that need to be changed every year, or a warning added at the top of the page that details might not be valid anymore.

File: source/gsoc/guide-to-being-a-mentor.md [hunk 2]
```diff
@@ -18,13 +18,13 @@ submit ideas via Jira (if your project does not use Jira you can [use the Comdev
...
-* Label the main issue with "*gsoc2023*" if appropriate (these will show up at [GSoC 2023 Ideas](https://s.apache.org/gsoc2023ideas))
+* Label the main issue with "*mentor*" (these will show up at the [ASF-wide list of issues](https://issues.apache.org/jira/issues?jql=labels%20in%20(gsoc2024)%20AND%20labels%20in%20(mentor,%20Mentor)))
+* Label the main issue with "*gsoc2024*" if appropriate (these will show up at [GSoC 2024 Ideas](https://s.apache.org/gsoc2024ideas))
 
 <div class="card border-success mb-3">
   <div class="card-header">Size of project</div>
   <div class="card-body text-success">
-    <p class="card-text">Starting this year there are 2 types of projects available.<br> Please put "<em>full-time</em>" label for ~350 hours project and<br> "<em>part-time</em>" label for ~175 hours project</p>
+    <p class="card-text">Starting this year there are 3 types of projects available.<br> Please put "<em>full-time</em>" label for ~350 hours project and<br> "<em>part-time</em>" label for ~175 hours project <br> "<em>small</em>" label for ~90 hours project</p>
```
Comment (bdelacretaz) [inline source/gsoc/guide-to-being-a-mentor.md:27]: I'd say "starting in 2024" so we're less likely to have stale info here later if we forget to update it ;-)

**Review by bdelacretaz — CHANGES_REQUESTED** [review]
Thanks for this, looks good to me with minor nitpicks added as comments.

File: source/gsoc/gsoc-admin-tasks.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ contributors to your projects.
 If you want to participate with your project you now need to
 
  - understand what it means to be a mentor [1]
- - propose your project ideas. Just label your issues with gsoc2023 in Jira and
+ - propose your project ideas. Just label your issues with gsoc2024 in Jira and
```
Comment (solomax) [inline source/gsoc/gsoc-admin-tasks.md:26]:
```suggestion
 - propose your project ideas. Just label your issues with `gsoc2024` in Jira and
```

**Review by solomax — APPROVED** [review]

**Review by bdelacretaz — APPROVED** [review]

**Comment by bdelacretaz** [comment]
Thanks @priyasharma1 ,all good for me!
I suspect the short code will change next January 1st, before GsoC 2025 is ready to go, but that's not dramatic...

**Pull request merged**